src/components/Table.js

```diff
@@ -283,9 +283,12 @@ export default class Table extends React.Component<TableDefaultProps, TableProps
       })
       : [];
       this.setState({ sortedRows }, () => {
-        const activeRow = sortedRows[this.state.activeRowIndex];
+        const activeRow = this.state.activeRowIndex !== null && newProps.noEmptySelection
```

Comment:
If the `noEmptySelection` flag is true, won't we end up with no selection if the previous activeRowIndex was null because there were no items, and now we have items? For example, if I filter out everything and then I delete the filter, should the first item not be selected/active?

Reply:
This is a good point. The same logic is used on [line 268](https://github.com/attivio/suit/pull/132/files#diff-7d12218e29d596148ef20ac652d3a3b9L268).